<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera_calibration_dialog</class>
 <widget class="QDialog" name="Camera_calibration_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="left_slider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>545</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>545</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1 mm =</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>545</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSlider" name="right_slider">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>545</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>545</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="suffix">
    <string> mm</string>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>34.609999999999999</double>
   </property>
  </widget>
  <widget class="QLabel" name="pixel">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>545</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>62.6 px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>545</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>O K</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>545</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>px =</string>
   </property>
  </widget>
  <widget class="QLabel" name="frame_width">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>545</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1920</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
